DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Azure/BlobHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using GitHubExtention.Preferences.WebApi.Constants;
+using Microsoft.WindowsAzure;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    internal static class BlobHelper
+    {
+        public static CloudBlobContainer GetWebApiContainer()
+        {
+            string str =
+                CloudConfigurationManager.GetSetting(AvatarConstants.StorageConnectionString);
+            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(str);
+
+            CloudBlobClient blobClient = storageAccount.CreateCloudBlobClient();
+            CloudBlobContainer container = blobClient.GetContainerReference(AvatarConstants.AzureContainer);
+            container.CreateIfNotExists();
+
+            var permissions = container.GetPermissions();
+            if (permissions.PublicAccess == BlobContainerPublicAccessType.Off)
```

Comment:
In the begining of request I check if the user found.
`string currentUserId = User.Identity.GetUserId();
            User user = await _userManager.Users.FirstOrDefaultAsync(x => x.Id == currentUserId);
            if (user == null)
                return NotFound();`
And if the user is not null than he has permissions, I guess. And I check permissions because it can be changed manually(out of code) in the azure account.